frontend/src/pages/secret-manager/SettingsPage/components/WorkflowIntegrationSection/components/AddWorkflowIntegrationModal.tsx

```
@@ -0,0 +1,154 @@
+              exit={{ opacity: 0, translateX: -30 }}
+            >
+              <div className="mb-4 text-mineshaft-300">Select a workflow integration</div>
+              <div className="flex items-center space-x-4">
+                {PLATFORM_LIST.map(({ icon, platform, title }) => {
+                  const isConfigured =
+                    (platform === WorkflowIntegrationPlatform.MICROSOFT_TEAMS &&
+                      microsoftTeamsConfigured) ||
+                    (platform === WorkflowIntegrationPlatform.SLACK && slackConfigured);
+
+                  return (
+                    <div className="relative">
```

Biome (1.9.4)

[error] Missing key property for this element in iterable.

The order of the items may change, and having a key can help React identify which item was moved.
Check the React documentation. 

(lint/correctness/useJsxKeyInIterable)